core/src/epicli/data/common/ansible/playbooks/roles/repository/defaults/main.yml

```
@@ -0,0 +1,5 @@
+teardown_full_repository: true
+teardown_helm_repository: false
+teardown_docker_images_repository: false
+teardown_packages_repository: false
+teardown_files_repository: false
```

```suggestion
teardown:
  disable_http_server: false
  remove:
    files: false
    helm_charts: false
    images: false
    packages: false
```
I would move these settings to common\defaults\configuration\repository.yml